src/System.IO.FileSystem/tests/File/EncryptDecrypt.cs

```diff
@@ -32,7 +32,7 @@ public static void EncryptDecrypt_NotSupported()
             Assert.Throws<PlatformNotSupportedException>(() => File.Decrypt(null));
         }
 
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer), nameof(PlatformDetection.IsNotWindowsHomeEdition))]
```

Comment:
Is that the right Syntax to check for two conditions?

Reply:
Please add a comment explaining why we skip the test on home editions.

Reply:
Yes it is the correct syntax